Tests/IntegrationTests/FastEndpoints.IntegrationTests/WebTests/MiscTestCases.cs

```diff
@@ -296,10 +296,10 @@ public async Task BindingFromAttributeUse()
                 "api/test-cases/route-binding-test/something/true/99/483752874564876/2232.12/123.45/" +
                 "?Bool=false&String=everything&XBlank=256" +
                 "&age=45&name=john&id=10c225a6-9195-4596-92f5-c1234cee4de7" +
-                "&numbers=0&numbers=1&numbers=-222&numbers=1000&numbers=22" +

```

Comment:
don't change this. we need to support this too.

Reply:
I know. It's still in progress ;)